source/includes/_coercion.md

```diff
@@ -0,0 +1,15 @@
+## coercion
+Change data type from one type to another. 
+
+Type coercion means that when the operands of an operator are different types, one of them will be converted to an "equivalent" value of the other operand's type
```

Comment:
Let's pitch this documentation at beginners.

What if they don't know what 'operands' and 'operators' are? Try to explain in the simplest terms possible.